clients/src/test/java/org/apache/kafka/clients/consumer/internals/FetchRequestManagerTest.java

```diff
@@ -781,10 +781,10 @@ public void testFetchSkipsBlackedOutNodes() {
         Node node = initialUpdateResponse.brokers().iterator().next();
 
         client.backoff(node, 500);
-        assertEquals(0, sendFetches());
+        assertEquals(1, sendFetches());
```

Comment:
Since we're not checking the node availablity using the networkClientDelegate in FetchRequestManager, the request that was made would pass through the check and made it way to unsentRequest. After the backoff the request should be sent already. 

Should I not change the test and make new one instead?